src/drm/drm.c

```diff
@@ -21,10 +21,6 @@
 #include "dtcmp.h"
 #include "mfu.h"
 
-/* TODO: change globals to struct */
-static int walk_stat = 0;
```

Comment:
With drm, it looks like we were avoiding the stat call by default.  I think it would be good to keep this, since that's easier on the file system.  Although I realize that means the --lite option is pointless, and we don't have an option to turn stat back on right now.

Reply:
@adammoody I updated to avoid stat by default, and I turn on the stat for the aggressive option. Yeah, the lite option is pointless now.